ServiceRegistryJpaImplTest.testIgnoreHostsInPriorityList regularly fails

Building the service registry regularly fails on its tests. I've seen this multiple times, both locally and on our test servers.

<details>

```
[INFO] Tests run: 2, Failures: 0, Errors: 0, Skipped: 0, Time elapsed: 0.116 s - in org.opencastproject.serviceregistry.impl.OsgiIncidentServiceTest                                                                                          
[INFO] Running org.opencastproject.serviceregistry.impl.ServiceRegistryJpaImplTest                                                                                                                                                            
20:55:45 ERROR (ServiceRegistryJpaImpl:639) - Job with Id 1 cannot be deleted: Not found.                                                                                                                                                     
[ERROR] Tests run: 12, Failures: 1, Errors: 0, Skipped: 0, Time elapsed: 19.328 s <<< FAILURE! - in org.opencastproject.serviceregistry.impl.ServiceRegistryJpaImplTest                                                                       
[ERROR] testIgnoreHostsInPriorityList(org.opencastproject.serviceregistry.impl.ServiceRegistryJpaImplTest)  Time elapsed: 2.125 s  <<< FAILURE!                                                                                               
java.lang.AssertionError                                                                                                                                                                                                                      
        at org.opencastproject.serviceregistry.impl.ServiceRegistryJpaImplTest.testIgnoreHostsInPriorityList(ServiceRegistryJpaImplTest.java:342)                                                                                             
                                                                                                                                                                                                                                              
[INFO] Running org.opencastproject.serviceregistry.impl.JobTest                                                                                                                                                                               
20:56:03  WARN (ServiceRegistryJpaImpl:551) - Creating a job from http://localhost:8080, which is currently in maintenance mode.                                                                                                              
20:56:03 ERROR (ServiceRegistryJpaImpl:1047) - originalJob.getProcessorServiceRegistration() is null                                                                                                                                          
[INFO] Tests run: 22, Failures: 0, Errors: 0, Skipped: 0, Time elapsed: 2.49 s - in org.opencastproject.serviceregistry.impl.JobTest                                                                                                          
[INFO] Running org.opencastproject.serviceregistry.impl.DispatchableComparatorTest                                                                                                                                                            
[INFO] Tests run: 7, Failures: 0, Errors: 0, Skipped: 0, Time elapsed: 0.086 s - in org.opencastproject.serviceregistry.impl.DispatchableComparatorTest                                                                                       
[INFO]                                                                                                                                                                                                                                        
[INFO] Results:                                                                                                                                                                                                                               
[INFO]                                                                                                                                                                                                                                        
[ERROR] Failures:                                                                                                                                                                                                                             
[ERROR]   ServiceRegistryJpaImplTest.testIgnoreHostsInPriorityList:342
```
</details>

--- a/modules/serviceregistry/src/test/java/org/opencastproject/serviceregistry/impl/ServiceRegistryJpaImplTest.java
+++ b/modules/serviceregistry/src/test/java/org/opencastproject/serviceregistry/impl/ServiceRegistryJpaImplTest.java
@@ -57,6 +57,7 @@ import org.easymock.IAnswer;
 import org.junit.After;
 import org.junit.Assert;
 import org.junit.Before;
+import org.junit.Ignore;
 import org.junit.Test;
 import org.osgi.framework.BundleContext;
 import org.osgi.framework.Filter;
@@ -315,6 +316,9 @@ public class ServiceRegistryJpaImplTest {
   }
 
   @Test
+  //Ignored because of https://github.com/opencast/opencast/issues/1281
+  //Long term this needs to be reenabled, but after a few months of trying I can't get it to reproduce locally :(
+  @Ignore
   public void testIgnoreHostsInPriorityList() throws Exception {
     if (serviceRegistryJpaImpl.scheduledExecutor != null)
       serviceRegistryJpaImpl.scheduledExecutor.shutdown();

